specs/DesignDiscussions/InputLabels.md

```diff
@@ -269,107 +251,26 @@ Similarly to option 1, a card author could use fallback to address this if they
-	- A placeholder with formatting wouldn't make sense, so if they add formatting to their label and then position it as the placeholder that formatting will be lost.
-2. Host decides via host-config (`"labelPosition": "Placeholder"`).
-    - Same issues as above if an explicit placeholder is set by the card author. On option for the case with both a label and a placeholder would be to only respect this if no inputs have placeholders (precedent from button icon placement). 
-3. Do nothing (in the schema)
-    - Card author can set placeholder instead of label if they want a placeholder to be the label. Undercuts our ability to reason over the card (we don’t know if the placeholder text is being used as the input’s label) but I’m not sure that has any practical negative consequences:
-        - Placeholder will serve as accessibility name in the absence of a label
-        - Are there responsiveness scenarios where we’d want to pull the label out of the placeholder to expand the card? (I’d guess not?)
-        - We don’t have a good place to put the “(optional)” indicator in this scenario. That said, the same will be true of any input that doesn’t use the features spec’d in this document to associate the labels.
-
-4. Use labels as placeholders only as part of a future responsiveness feature. If we're condensing the space of the card, and there are no placeholders, consolidating labels into placeholders would be one way to adapt to limited space. This could be combined with 1, 2, or 3.
-
-As there doesn't seem to be a strong reason to do so, for now, we will not make any changes to allow card authors to indicate that their placeholders are actually labels.
-
-## Label Position
-As alluded to above, we should consider how the label position is determined. One of the goals of this feature is to allow us to someday intelligently reflow inputs, so it may be the case that we want to maintain control of label positioning. Today we would always draw the label above the input, but in the future we may put it to the side if our rendering space is wide enough, or push it into placeholders if our space is constrained.
-
-That said, it may be desirable let either the card author or the host configure the position of the label. The tradeoff is more layout power to the card author, and less flexibility to us to reflow in the future. Because of this, we will not introduce this with v1 of this feature, and consider it only if we get customer feedback or requests to allow positioning of labels.
-
-##### Label Above
+We could also possibly add a specific named feature to the requires dictionary to allow card authors to address it directly using something like:
 ```json
-{
-	"type": "Input.Text",
-	"id": "nameInput",
-	"label": "Please enter your name:",
-	"labelPosition": "Above",
-	"placeholder": "Last, First"
+"requires": {
+	"inputLabels": "1.0"
 }
```

Comment:
Has there been any further discussion/decision around whether we want to do this? If we're not planning to, we should probably say that here or remove this idea from the spec.